docs/snapshotting/snapshot-support.md

```diff
@@ -87,7 +88,8 @@ target is under **8ms** with 5ms p90 for a microvm with this specs:
 
 - High snapshot latency on 5.4+ host kernels - 
 [#2129](https://github.com/firecracker-microvm/firecracker/issues/2129)
-- Guest network connectivity is not guaranteed to be preserved after resume
+- Guest network connectivity is not guaranteed to be preserved after resume.
+For ensuring guest network connectivity please see [Network connectivity for clones](network-for-clones.md)
```

Comment:
I would make it clear that active network connections cannot be guaranteed (even with the doc):

```suggestion
- Guest network connectivity is not guaranteed to be preserved after resume.
For ensuring guest network connectivity please see [Network connectivity for clones](network-for-clones.md). Active network connections are not guaranteed to survive a snapshot cycle.
```

Reply:
I modified the wording. PTAL